obelisk_ws/src/obelisk_ros/config/dummy_cpp.yaml

```diff
@@ -3,6 +3,7 @@ onboard:
   control:
     - pkg: obelisk_control_cpp
       executable: example_position_setpoint_controller
+      params_path: /path/to/my/file
```

Comment:
would be better to actually update the dummy example to use this in some example way. for example, just specifying a param file with a single param and having that somehow be used